rebasehelper/archive.py

```diff
@@ -182,6 +182,23 @@ def extract(cls, archive=None, filename=None, path=None, *args, **kwargs):
 
 
 @register_archive_type
+class AscArchiveType(ArchiveTypeBase):
+    """ .asc files are not archives - this is a PGP file type"""
+    EXTENSION = ".asc"
+
+    @classmethod
+    def open(cls, filename=None):
+        pass
+
+    @classmethod
+    def extract(cls, archive=None, filename=None, path=None, *args, **kwargs):
+        if archive is not None:
+		        raise RuntimeError("In Asc files, the archive (pos 1) argument is not used, but passed")
```

Comment:
There are two tab characters on this line, please replace them with spaces. Thanks.